website_chd_product_configurator/controllers/controllers.py

```diff
@@ -0,0 +1,237 @@
+# -*- coding: utf-8 -*-
+from openerp import http
+from openerp.http import request
+from openerp.osv import orm
+import json
+
+
+class Chd_website(http.Controller):
+
+    @http.route('/chd_init/', auth='public', website=True)
+    def start(self, selected_id=False, type=False):
+        partner = self.get_current_partner()
+        if partner is False:
+            return request.render(
+                'website_chd_product_configurator.no_partner', {})
+        Conf_products = request.env['product.template']
```

Comment:
invent some of the other standard to name references to models, so that we can infer from the variable name which model it is. People use `model_name_obj` for example. I personally use `full_model_name` and name the recordsets differently